apps/client/src/components/AddressLine.tsx

```diff
@@ -14,9 +15,11 @@ export default ({
   streetName,
 }: AddressLineProps) => (
   <>
-    {streetName} {houseNumberFull}
-    <div>
+    <Paragraph gutterBottom={0}>
+      {streetName} {houseNumberFull}
+    </Paragraph>
+    <Paragraph gutterBottom={16}>
```

Comment:
Somehow this component morphed from an addresline into AddresLines (plural). Can you update the filename to [reflect these changes](https://github.com/Amsterdam/vergunningcheck/pull/556/files)?
Also please add tests for `{postalCode} {residence}` if you have the time.

Reply:
There are already test for postalCode and Residence